strider/knowledge_provider.py

```diff
@@ -47,6 +48,8 @@ def __init__(
             logger=logger,
             preproc=self.get_preprocessor(kp["details"]["preferred_prefixes"]),
             postproc=self.get_postprocessor(WBMT.entity_prefix_mapping),
+            max_batch_size=1,
```

Comment:
Have we thoroughly tested how not batching will impact performance?

Reply:
Good catch. This snuck in cause I have other changes for batch sizing. I've removed it.